nil/internal/execution/fee.go

```diff
@@ -67,11 +83,21 @@ func calculateBaseFee(baseFeePrevious types.Value, gasUsedPrevious types.Gas) ty
 	baseFeePreviousFixed := decimal.NewFromBigInt(baseFeePrevious.Int().ToBig(), 0)
 	newFee := baseFeePreviousFixed.Mul(adjustmentFactor.Mul(normalized).Add(decimal.New(1, 0)))
 
+	var resultBaseFee types.Value
```

Comment:
Then you don't need else branch
```suggestion
	resultBaseFee := types.DefaultGasPrice
```